Annotations.md

```diff
@@ -0,0 +1,134 @@
+# Acquia RA composer.json Template w/Annotations
+[Name and Description](#name-and-description)
+[Type](#type)  
+[License](#license)  
+[Repositories](#repositories)  
+[Require](#require)  
+[Require-Dev](#require-dev)  
+[Conflict](#conflict)  
+[Minimum-Stability](#minimum-stability)  
+[Prefer-Stable](#prefer-stable)  
+[Extra](#extra)  
+[Installer-Paths](#installer-paths)  
+[Scripts](#scripts)  
+[Patches](#patches)  
+#### Name and Description
+```
+{
+    "name": "acquia/acquia-ra-composer",
+    "description": "Project json for Acquia Remote Administration Composer builds",
+```
+This is the name of your project and a brief description of what your project consists of.
+#### Type
+```
+    "type": "project",
+```
+Types are used througout your composer.json. You can find more information here:
+[composer.org: Schema](https://getcomposer.org/doc/04-schema.md#type)
+#### License
+```
+    "license": "proprietary",
+```
+You can specify the type of license that is most appropriate to for your project. You can review additional license types at  getcomposer.org:
+[composer.org: License](https://getcomposer.org/doc/04-schema.md#license)
+#### Repositories
+```
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    },
+```
+By default, Composer will look for packages on packagist.org. You can add additional repositories to check against in this section:
+[composer.org: Repositories](https://getcomposer.org/doc/04-schema.md#repositories)
+#### Require
+```
+    "require": {
+        "composer/installers": "^1.0",
+        "cweagans/composer-patches": "^1.6.0",
+        "drupal-composer/drupal-scaffold": "^2.0.0",
+        "drupal/admin_toolbar": "^1.0",
+        "drupal/core": "^8.4",
+        "drush/drush": "^9.0"
```

Comment:
should we be making 9 the default until ENT-1984 is addressed? or do i have my RA wires crossed.

Reply:
You're probably right and that's probably prudent.

I'll wait to squash and merge until that ticket is complete.